qxf2_scheduler/templates/get-schedule.html

```diff
@@ -120,26 +125,57 @@
                         }
                         else {
                             if (freeChunkSlots.length == 0) {
-                                msg += 'There were no free slots for ' + result.email + ' on ' + result.date
+                                msg += 'There were no free slots for ' + result.email + ' on ' + result.date;
                             } else {
-                                msg = 'The free slots for ' + result.email + ' on ' + result.date + ' are: <br>'
-                                var elements = [];
+                                msg = 'The free slots for ' + result.email + ' on ' + result.date + ' are: <br>';
                                 for (index in freeChunkSlots) {
-                                    var timeSlotButton = $('<div class="col-md-2 top-space-20"><input type="button" class="btn" value="' + freeChunkSlots[index].start + ' - ' + freeChunkSlots[index].end +'"></div>');
-                                    elements.push(timeSlotButton);                                                                              
-                                }                                
+                                    slot = freeChunkSlots[index].start + ' - ' + freeChunkSlots[index].end;
+                                    var timeSlotButton = $('<div class="col-md-2 top-space-20"><input type="button" onclick="displayChosenSlot(\'' + email + '\',\'' + date + '\',\'' + slot + '\')" class="btn" value="' + slot + '"></div>');
```

Comment:
Can you please verify the button styling here? When I opened it in Chrome, the button seemed invisible. I believe either the CSS class used is incorrect or we need to add a color to the button.

Reply:
I have added a 'info' color to the button